test/with_api_v2/acceptance/api_test.go

```diff
@@ -89,8 +89,8 @@ receivers:
 	require.NoError(t, err)
 	// No silence has been created or inhibiting alert sent, alert should
 	// be active.
-	for _, alert := range resp.Payload {
```

Comment:
There's a couple more like this but these it's just fixing package names matching variables.